### DISCUSSION TITLE
How to get DateTime less than milliseconds?

### DISCUSSION THREAD

**Discussion by Jun-Murakami** [supabase-community/supabase-csharp#80] [Q&A]
Hi, Thanks for developing this very useful library.

Storing DateTime type values in the database,
when I fetch it locally, the decimal second values are lost. （milliseconds/microseconds）

I don't know if it's a bug, my usage is wrong, or a specification, so I'm posting it in this forum.

**Comment by acupofjose** [comment]
As far as I can tell @Jun-Murakami, PostgREST (the database API that's leveraged for this library) doesn't have support for timestamps with milli/microseconds (at least it's not listed [in their API documentation](https://postgrest.org/en/stable/how-tos/working-with-postgresql-data-types.html?highlight=timestamp#timestamps)). An alternative would be to store the value as a `bigint` on the database and do the conversion from a timestamp to a `DateTime` object.

**Reply by Jun-Murakami** [reply]
Good to know that. I will try to operate with other data types as you suggested. Thanks a lot.
